templates/app-autoscaler-deployment.yml

```diff
@@ -141,7 +142,7 @@ instance_groups:
   instances: 1
   update:
     serial: true
-  stemcell: default
+  stemcell: xenial
```

Comment:
maybe we should use default and see if the acceptance tests fail ?

Reply:
```suggestion
  stemcell: default
```